cmd/utils/selector.go

```diff
@@ -38,6 +39,9 @@ const (
 	showCursor = esc + "?25h"
 )
 
+// ErrInvalidOption is raised when the selector has an invalid option
```

Comment:
why not in errors pkg?

Reply:
Let's start adding errors in the package that creates them, that way we can do `selector.ErrInvalidOption`, which provides to know which package is raising the error instead of having a general errors package